data/test/scenarios/ability_id_active_test.cfg

```diff
@@ -1,3 +1,15 @@
+#####
```

Comment:
Please put the `# wmllint: no translatables` in this file on the first line too.

I'm okay with not adding it if a file doesn't already have it, because that would require checking whether it's accurate or not. I just want to ensure that any new comments are added below it.